### DISCUSSION TITLE
What does your favorite Postgres client do really well?

### DISCUSSION THREAD

**Discussion by aleclarson** [alloc/tusken#28] [General]
Which Postgres client have you been using? What do you really like about it?

**Note: I'm only asking about query builders, not ORMs or plain SQL clients.** 😄

**Comment by ViktorQvarfordt** [comment]
I'm looking for a TS version of jOOQ: Typesafe DSL for SQL. Looking for 1:1 support with SQL, not looking for abstractions on top.

**Reply by aleclarson** [reply]
Have you tried [Drizzle](https://github.com/drizzle-team/drizzle-orm)?

**Reply by ViktorQvarfordt** [reply]
Thanks for the tip!
